api4/channel_test.go

```diff
@@ -1837,9 +1837,6 @@ func TestRemoveChannelMember(t *testing.T) {
 		t.Fatal("should have passed")
 	}
 
-	_, resp = Client.RemoveUserFromChannel(th.BasicChannel.Id, th.BasicUser2.Id)
-	CheckNoError(t, resp)
-
```

Comment:
The same user / channel combination was attempted to deleted twice, which seems to be a mistake. The code before the introduced hook changes did not actually check whether a Channel Membership existed, I've removed the second test call in question, happy to discuss that of course.